package.json

```diff
@@ -31,7 +31,7 @@
     "vuepress": "^1.0.2"
   },
   "dependencies": {
-    "@rei/cedar": "^3.0.0-alpha.2",
+    "@rei/cedar": "^3.0.0-accordion.alpha.1",
```

Comment:
3.0.0-alpha.2 already had the accordion updates in it. Does your alpha have my v-model changes in it?

Reply:
Yeah, sorry for the confusion here. I found a bug in the accordion code while trying to update the nav with `alpha.2`. That PR can be found here: https://github.com/rei/rei-cedar/pull/662